### ISSUE TITLE
Checksum error in glyf

### ISSUE THREAD

**Issue by Th3S4mur41** [L2jLiga/grunt-webfonts#7]
I'm getting the following error when trying to generate ttf font [icon-call-window-front.zip](https://github.com/L2jLiga/grunt-webfonts/files/2491884/icon-call-window-front.zip) and the node engine

```
\node_modules\ttf2woff2\jssrc\ttf2woff2.js:1
(function (exports, require, module, __filename, __dirname) { var Module;if(!Module)Module=(typeof Module!=="undefined"?Module:null)||{};var moduleOverrides={};for(var key in Module){if(Module.hasOwnProperty(key)){moduleOverrides[key]=Module[key]}}var ENVIRONMENT_IS_WEB=typeof window==="object";var ENVIRONMENT_IS_WORKER=typeof importScripts==="function";var ENVIRONMENT_IS_NODE=typeof process==="object"&&typeof require==="function"&&!ENVIRONMENT_IS_WEB&&!ENVIRONMENT_IS_WORKER;var ENVIRONMENT_IS_SHELL=!ENVIRONMENT_IS_WEB&&!ENVIRONMENT_IS_NODE&&!ENVIRONMENT_IS_WORKER;if(ENVIRONMENT_IS_NODE){if(!Module["print"])Module["print"]=function print(x){process["stdout"].write(x+"\n")};if(!Module["printErr"])Module["printErr"]=function printErr(x){process["stderr"].write(x+"\n")};var nodeFS=require("fs");var nodePath=require("path");Module["read"]=function read(filename,binary){filename=nodePath["normalize"](filename);var ret=nodeFS["readFileSync"](filenam
**Checksum error in glyf**

Process finished with exit code 7

```

Any idea what is wrong with this SVG? I only see the error with few SVG icons, but cannot identify a common pattern I could identify as root cause.
Generating icon fonts with the same SVG works fine in Icomoon

Also interesting fact, the task is not throwing any error if the list of files contains at least one working 
SVG. So until you check each icon in the font, you get the impression that everything went well, which is not true.

**Comment by L2jLiga** [comment]
@Th3S4mur41 after some research I found that this happens with provided icon because it contains `rect` with `rx` attribute (after removing this attr all works fine),
Currently I have no any solution for that, will looking for it..

---
**Upd:** released version 1.1.2 under `next` tag in npm with fix

**Comment by Th3S4mur41** [comment]
Works like a charm now. Thanks
